src/Target.cpp

```diff
@@ -62,6 +62,63 @@ static void cpuid(int info[4], int infoType, int extra) {
 #endif
 #endif
 
+#if defined(__x86_64__) || defined(__i386__) || defined(_MSC_VER)
```

Comment:
Not necessarily as part of this PR, but... this file is getting a lot of processor-specific ifdef stuff in it, which is getting to be a code smell at this point IMHO (I'm a big believer in trying to keep "core" code mostly-core and segregating processor/platform/etc specific code into separate units where possible). We should think about whether and/or how to reorganize this (after these changes) to keep this from exploding into a mess of messiness.

Reply:
Makes sense to me.